### ISSUE TITLE
To make the tor-0.2.6.1-alpha code working you will need and old version of openssl

### ISSUE THREAD

**Issue by brammittendorff-dd** [0x3a/tor-dns#2]
So first off you will need to download an old version of openssl:

https://ftp.openssl.org/source/old/1.0.1/

```
tar zxvf openssl*
```

Compile for example with:

```
./config enable-ec_nistp_64_gcc_128 shared zlib-dynamic --prefix=/usr/local/openssl --openssldir=/usr/local/openssl
make
make install
```

Compile the tor-0.2.6.1-alpha:

```
./configure --with-openssl-dir=/usr/local/openssl --enable-static-openssl
make
make install
```

Now you can use the patched tor:
```
./tor -v
Sep 04 16:37:19.243 [notice] Tor v0.2.6.0-alpha-dev running on Linux with Libevent 2.0.21-stable, OpenSSL 1.0.1u and Zlib 1.2.8.
Sep 04 16:37:19.243 [notice] Tor can't help you if you use it wrong! Learn how to be safe at https://www.torproject.org/download/download#warning
Sep 04 16:37:19.243 [notice] This version is not a stable Tor release. Expect more bugs than usual.
Sep 04 16:37:19.243 [warn] Command-line option '-v' with no value. Failing.
Sep 04 16:37:19.243 [err] Reading config failed--see warnings above.
```

**Comment by 0x3a** [comment]
Dependencies are listed in the configure script from the Tor source: https://github.com/0x3a/tor-dns/blob/master/patched%20tor/tor-0.2.6.1-alpha/configure

This is not an issue, closing.

**Comment by brammittendorff-dd** [comment]
You are right my bad.
